app/Http/Controllers/LiquidationController.php

```
@@ -0,0 +1,81 @@
+        $validatedData = $request->validated();
+        $stakeholder = StakeHolder::where('name', $validatedData['project_code'])->first();
+        $validatedData['stakeholder_id'] = $stakeholder?->id ?? null;
+        $validatedData['prf_no'] = 'LIQ-' . uniqid();
+        $validatedData['type'] = PaymentRequestType::LIQUIDATION->value;
+        $validatedData['created_by'] = auth()->user()->id;
+        $validatedData['request_status'] = RequestStatuses::PENDING->value;
+        $validatedData['description'] = $validatedData['project_code'];
+        $validatedData['total_vat_amount'] = 0;
+        $validatedData['total'] = $validatedData['amount'];
+
+        $paymentRequest = PaymentRequest::create($validatedData);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\PaymentRequest' in method 'store'. (undefined)

(StaticAccess)